README.md

```
@@ -1,6 +1,31 @@
 # Install and Run
+
+## Installation
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)